### PR TITLE
Removed unread private members

### PR DESCRIPTION
**Pull request by Mailaender**
https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0052

### CODE REVIEW

**Review by RoosterDragon — APPROVED** [review]

File: OpenRA.Mods.Cnc/UtilityCommands/ImportTSMapCommand.cs [hunk 1]
```diff
@@ -239,9 +239,6 @@ class ImportTSMapCommand : IUtilityCommand
 			{ 0x03, new byte[] { 0x7E } }
 		};
 
-		// Veins and vein holes
-		static readonly int[] ValidVeinNeighbours = { 0x7E, 0xA7, 0xB2 };
```
Comment (PunkPun) [inline OpenRA.Mods.Cnc/UtilityCommands/ImportTSMapCommand.cs:243]: This was added in #19090. Are we sure we aren't losing something important by removing it?

Comment (Mailaender) [reply OpenRA.Mods.Cnc/UtilityCommands/ImportTSMapCommand.cs:243]: @pchote Is this for future use?

Comment (penev92) [reply OpenRA.Mods.Cnc/UtilityCommands/ImportTSMapCommand.cs:243]: The relevant bit is  https://github.com/OpenRA/OpenRA/commit/fcc3008b00fe58b3b3111efa3e7526f5b4f85048#diff-1b46bd8993ce1ce1b586eb8f62f370625423f2f13d779e61293e88fba0825527R241-R466  but I don't know what to make of it.

Comment (Mailaender) [reply OpenRA.Mods.Cnc/UtilityCommands/ImportTSMapCommand.cs:243]: It seems only `0x7E` is used, and it is referenced elsewhere. This is dead code.

**Review by PunkPun — APPROVED** [review]

**Comment by penev92** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/629a4d0643ecbeea711829e95d9b8cc944e8d0f4...1717645e8acf94ad3117d0812dd4fbdf2f21df04)

**Pull request merged**